### ISSUE TITLE
Stereoscopic 3D view not working on R7

### ISSUE THREAD

**Issue by alex-amaral** [ODG-Github/OdgUnityPlugin#1]
The stereoscopic 3D seems no work. It looks like a normal 2D view, similar when looking to a tablet.

I'm testing on a R7

**Comment by danilogr** [comment]
Hey @alex-amaral, what's the version of your Unity editor?

**Comment by ODGTeam** [comment]
You'll have to adjust the left and right cameras. If it still doesn't look stereo I suggest using a script to adjust the cameras during runtime.

![left](https://user-images.githubusercontent.com/16826527/35823718-50261048-0a65-11e8-8590-ac375c24385c.jpg)
![right](https://user-images.githubusercontent.com/16826527/35823722-544b35b8-0a65-11e8-9bdc-39bfac6fc387.jpg)

**Comment by danilogr** [comment]
Isn't that what `OdgStereoCameraPlugin.cs` does?

**Comment by ODGTeam** [comment]
Yes the script will need to be changed. The current defaults doesn't match the R-7s convergence distance and FOV. So you won't get much of a stereo effect. We will update the plugin.
